src/community/oseo/oseo-stac/src/main/java/org/geoserver/ogcapi/v1/stac/STACService.java

```diff
@@ -525,6 +537,46 @@ public SearchResponse searchPost(@RequestBody STACSearchQuery sq) throws Excepti
 
         return response;
     }
+    /**
+     * Based on STAC API Extension for Sort
+     * https://github.com/stac-api-extensions/sort?tab=readme-ov-file#http-post-json-entity *
+     */
+    private SortBy[] stacSortByToGTSortby(SortBy[] sortBy) {
```

Comment:
I'd rather have the custom parsing in ``STACSearchQuery#setSortBy(JsonNode)``

Reply:
Done